wordpress-mobile/WordPress-iOS
# Update pod to WordPressAuthenticator-iOS version 1.0.5

In this PR we update to the WordPressAuthenticator-iOS version 1.0.5 release candidate. 

**Expected Behavior: 
There should be no visual changes to the WordPress app.** 

The authenticator changes exposed the login screen background colors and the navigation logo for styling in the host apps. The default color for `viewControllerBackgroundColor` is set to `WPStyleGuide.lightGrey()` and the default logo for the `navBarImage` is our WordPress gridicon.

**Note:** It doesn't matter which login flow you choose test, since the view controller background color change happens on the base class in WPAuthenticator, `LoginViewController`.

## To Test
-  check out this PR and run `pod install`.
-  (log out of the app if you are logged in). 
- follow the login flows 

Original background styling / WPAuth background styling with defaults
<img src="https://user-images.githubusercontent.com/1062444/43726830-d3ab381e-9965-11e8-9b66-dea6cbe873f3.png" width="250" align="left" /> <img src="https://user-images.githubusercontent.com/1062444/43726859-e7580752-9965-11e8-9c45-38de5b2584b5.png" width="250" align="left" />
